src/esw/fw/bdcmc/Core/Inc/motion_profile.hpp

```diff
@@ -0,0 +1,59 @@
+#include "units/units.hpp"
+
+namespace mrover {
+    template<IsUnit PositionUnit = Radians, IsUnit TimeUnit = Seconds>
```

Comment:
Hey at least someone other than me understands how to use the units library! Cool cool